test-data/unit/check-flags.test

```diff
@@ -1099,26 +1099,26 @@ def f(c: C) -> None:
 [out]
 
 [case testCheckDisallowAnyGenericsAnyGeneric]
-# flags: --disallow-any-generics
+# flags: --new-semantic-analyzer --disallow-any-generics
 from typing import TypeVar, Callable
 
 T = TypeVar('T')
 C = Callable[[], T]
 
-def f(c: C):  # E: Missing type parameters for generic type
+def f(c: C):  # E: Missing type parameters for generic type "Callable[[], T?]"
```

Comment:
A tiny bit of bikeshedding: it would be better to show original alias name (if available at this point) instead of the expansion. But this is totally optional, since this is already much better than the current state.